src/event_handler.py

```diff
@@ -11,15 +11,22 @@
 
 logging.basicConfig(level=logging.INFO)
 
-
 # noinspection PyUnusedLocal
 def store_queued_events(_, __):
     sqs_client = boto3.client('sqs')
     queue_url = os.environ['QUEUE_URL']
-    db_connection = create_db_connection()
-    encrypted_decryption_key = fetch_decryption_key()
+
+    if 'ENCRYPTION_KEY' in os.environ:
+      encrypted_decryption_key = os.environ['ENCRYPTION_KEY']
+    else:
+      encrypted_decryption_key = fetch_decryption_key()
```

Comment:
I think you can do `encrypted_decryption_key = os.environ.get('ENCRYPTION_KEY', fetch_decryption_key())`